training/training.pages.inc

```diff
@@ -0,0 +1,222 @@
+<?php
+/**
+ * @file
+ * Theme file for training module.
+ */
+
+/**
+ * Rendering example form.
+ *
+ * @param array $arg
+ *  Id of example form.
+ *
+ * @return array|mixed
+ *   The form array.
+ */
+function training_example_form_callback($arg) {
+  return drupal_get_form($arg);
+}
+
+/**
+ * Rendering list of links on other theme pages.
+ *
+ * @return string
+ *   String contents HTML tags for custom layout.
+ */
+function training_theme_html() {
+  $array = array(
+    '<a href="theme/list">' . t('List') . '</a>',
```

Comment:
use l() for building a link

https://api.drupal.org/api/drupal/includes%21common.inc/function/l/7